src/styles/cylc/_drawer.scss

```diff
@@ -20,13 +20,13 @@
 
 body.resizing-drawer {
   cursor: ew-resize !important;
+  #c-sidebar, .v-main {
+    // Prevent Vuetify-provided transitions during resize to ensure responsiveness
+    transition: none !important;
+  }
 }
 
 #c-sidebar {
-  /* this is not in our styles/material-dashboard, so we need to force-override */
-  -webkit-box-shadow: none !important;
-  box-shadow: none !important;
```

Comment:
I think this was outdated, wasn't doing anything

Reply:
There is a visible drop shadow when opening the draw in the workspace view from its collapsed state (i.e. via the toggle button in the top left), but this is fine.